pkg/node/volumemgr.go

```diff
@@ -582,6 +583,9 @@ func (m *VolumeManager) Discover() error {
 	drivesResponse, err := m.driveMgrClient.GetDrivesList(ctx, &api.DrivesRequest{NodeId: m.nodeID})
 	driveMgrDoneFunc()
 	if err != nil {
```

Comment:
We probably still need to know that there was some sort of error in GetDrivesList function even though status.FromError fails.